backend/dal/Helpers/Extensions/PropertyExtensions.cs

```diff
@@ -70,6 +70,8 @@ public static IQueryable<Entity.Views.Property> GenerateQuery(this PimsContext c
                 query = query.Where(p => EF.Functions.Like(p.ProjectNumber, $"{filter.ProjectNumber}%"));
             if (filter.IgnorePropertiesInProjects == true)
                 query = query.Where(p => p.ProjectNumber == null);
+            if(filter.ProjectPropertiesOnly == true)
+                query = query.Where(p => p.ProjectNumber.Contains("SPP"));
```

Comment:
this is what I am mainly wondering about, is this sufficient?

Reply:
I think the following is more correct.  Generally returning anything with project number and has been approved.
`!String.IsNullOrWhitespace(Property.ProjectNumber) && !property.Projects.Any(p => p.IsProjectInDraft() || !p.IsProjectClosed())`